src/parser.mjs

```diff
@@ -3,16 +3,16 @@ import {LRParser} from "@lezer/lr"
+  stateData: "8i~O!YOSPOS!WPQ~O!_QO!hRO!kSO~O!WXO~O![YO!]YO~O#S^O~OP`O!Y`O![YO!]YO~O!UcO~O!UeO~O!UgO~O!alO!blO!clO~O!eoO!ioO~O!atO!esO!isO!ltO!mtO!ntO~O!UzO~O![YO!]YO!d!OO~O!V!QO!]!QO!alO!blO!clO~O![TX!]TX!dVX!jTX~O!d!RO~O![YO!]YO!j!UO~O!V!WO!]!WO!eoO!ioO~O![TX!]TX!dVX!eTX!iTX!oTX!tTX~O!e!YO!i!YO!o!ZO!t![O![[X!][X~O!esO!isO~O!V!cO!]!cO~P!|O!e!dO!i!dO#U!eO#V!iO#Y!fO#[!gO#i!jO#j!lO#k!lO~O!UzO!Tji~O!e!sO!f!sO~O!e!tO!i!tO~O!d!RO![Ua!]Ua!jUa!fUa!wUa#]Ua#^Ua#_Ua#bUa#cUa#dUa#eUa#fUa#gUa#hUa#WUa#`Ua#aUa~O!e!wO!i!wO~O!d!RO![Ua!]Ua!eUa!iUa!oUa!tUa~O!p!yO~O!e!zO!i!zO!v!{O~O!t![O![[a!][a~O!o!ZO!t![O![[a!][a~O!e!YO!i!YO!o!ZO!t![O![[a!][a~O!U#SO!V}a!]}a!a}a!e}a!i}a!l}a!m}a!n}a~O![tX!]tX!dVX!ftX!wtX#]tX#^tX#_tX#btX#ctX#dtX#etX#ftX#gtX#htX#WtX#`tX#atX~O#V#UO~O#V#VO~O#V#WO~O!e#YO!i#YO#V!iO#j#ZO#k#ZO~O#]#]O#^#]O#_#]O#`#]O#a#]O~O!f#_O!w#_O#]#aO#^#aO#_#`O#b#_O#c#_O#d#`O#e#`O#f#`O#g#`O#h#`O~O![YO!]YO~P-rO!V#hO!]#hO~P&bO!q#kO~O!e#mO~O!e#oO!i#oO!q#oO!t![O!v#qO!w#pO!y#nO!z#nO!{#oO!|cX!}cX~O!|#sO!}!uX~O!}#uO~O!t![O![[i!][i~O!o!ZO!t![O![[i!][i~O!p#xO#O#yO#P#yO#Q#yO#R#yO~O#W$PO~P-rO![ta!]ta!dVX!fta!wta#]ta#^ta#_ta#bta#cta#dta#eta#fta#gta#hta#Wta#`ta#ata~O!q$SO!w$TO#V!iO~O!q$SO!w$TO~P&bO!e!dO!i!dO!q$SO!w$TO#V!iO#i!jO#j!lO#k!lO~O!r$[O~O!v$]O~O!q$^O!{$^O~O!e$_O~O!|#sO!}!ua~O!t![O![[q!][q~O!V$gO!]$gO~P0uO#W$hO~O#W$iO~O#W$jO~O!q$kO~O!e#YO!i#YO#j#ZO#k#ZO~O#`#]O#a#]O![li!]li!fli!wli#]li#^li#_li#bli#cli#dli#eli#fli#gli#hli#Wli~O!q$oO~O!e$_O!v$rO~O!r$sO~O!f#_O!w#_O#b#_O#c#_O~O!e!dO!i!dO#V!iO#i!jO#j!lO#k!lO~O#W$PO~O!s$xO~O!s${O~O#O#P#Q#R#S!o#U#Y!l!n!m!a#[!w!z!y!{!q!e#k#j!m~",
```

Comment:
I guess this is from the parser generator? Isn't this generated by `@lezer/generator`? Why is it committed?